src/project/git/GitCommandGitProject.ts

```diff
@@ -292,11 +301,14 @@ export class GitCommandGitProject extends NodeFsLocalProject implements GitProje
  * @param opts options for clone
  * @param directoryManager strategy for cloning
  */
-function clone(credentials: ProjectOperationCredentials,
-               id: RemoteRepoRef,
-               opts: CloneOptions,
-               directoryManager: DirectoryManager,
-               secondTry: boolean = false): Promise<GitProject> {
+function clone(
+    credentials: ProjectOperationCredentials,
+    id: RemoteRepoRef,
+    opts: CloneOptions,
+    directoryManager: DirectoryManager,
+    secondTry: boolean = false,
+): Promise<GitProject> {
+
```

Comment:
Why that change? Just looks like formatting. Maybe we need some agreement how we want that to be formatted.

Reply:
Both good reasons. Thanks.